klayout/drc/rule_decks/main.drc

```diff
@@ -1019,17 +904,16 @@ def conn_space(layer,conn_val,not_conn_val, mode)
     end
   end
   unconnected_output = unconnected_errors.polygons.or(singularity_errors.polygons(0.001))
-  return connected_output, unconnected_output
+  [connected_output, unconnected_output]
 end
 
-def conn_separation(layer1, layer2, conn_val,not_conn_val, mode)
-  if conn_val > not_conn_val
-    raise "ERROR : Wrong connectivity implementation"
-  end
+def conn_separation(layer1, layer2, conn_val, not_conn_val, mode)
```

Comment:
is this function used?

Reply:
Yes, in other tables. 'main' is for all tables.

Reply:
can we add it once it's actually used, otherwise it won't be covered by any tests.

Reply:
@proppy I prefer not to change `main` in every PR to make the PRs easier to merge. That's why I kept it here. And it's covered in the tables that uses this function.

Reply:
sure, as long as it's covered in the other table I guess it's fine!